hyperjumptech/monika
# Fixed(Config): write config file from har or postman

# Monika PR
fix #282 Generate monika.json from a HAR file
## What issue this PR is trying to solve
Need to add capability to generate monika.json config file from har or postman
## How this PR solve the issue
By writing to file the configuration parsed from har or postman if it has the flag `--create-config` while parsing from `--har` or `--postman` file flags.
## How to test
1. run `./bin/run --create-config --har ./test/testConfigs/harTest.har --output monika.config.json`
2. check on base folder `monika.config.json` file should be created
